rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -60,13 +60,18 @@ def test_exec(config, ssh_con):
     rgw_service = RGWService()
 
     ip_and_port = s3cmd_reusable.get_rgw_ip_and_port(ssh_con)
+    if config.haproxy:
+        hostname = socket.gethostname()
+        ip = socket.gethostbyname(hostname)
+        port = 5000
+        ip_and_port = f"{ip}:{port}"
 
     # CEPH-83575477 - Verify s3cmd get: Bug 2174863 - [cee/sd][RGW] 's3cmd get' fails with EOF error for few objects
     if config.test_ops.get("s3cmd_get_objects", False):
         log.info(f"Verify 's3cmd get' or download of objects")
         user_info = resource_op.create_users(no_of_users_to_create=config.user_count)
         s3_auth.do_auth(user_info[0], ip_and_port)
-        auth = Auth(user_info[0], ssh_con, ssl=config.ssl)
+        auth = Auth(user_info[0], ssh_con, ssl=config.ssl, haproxy=config.haproxy)
```

Comment:
@viduship , wouldn't this fail, in case haproxy is not configured.

Reply:
no it will not fail even if no haproxy is present.